src/content/peerconnection/constraints/js/main.js

```diff
@@ -290,10 +290,9 @@ setInterval(function() {
 // might be named toString?
 function dumpStats(results) {
   var statsString = '';
-  Object.keys(results).forEach(function(key, index) {
-    var res = results[key];
+  results.forEach(function(res) {
     statsString += '<h3>Report ';
-    statsString += index;
+    statsString += res.type;
```

Comment:
hah. I wanted to point out that change. I don't think it makes sense to have a numeric index here. I tried the report id but that got very long and ugly